src/module.api/anchors.controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Controller, Get, Query } from '@nestjs/common'
+import { ListAnchorsResult } from '@defichain/jellyfish-api-core/dist/category/spv'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { AnchorData } from '@whale-api-client/api/anchors'
+import { PaginationQuery } from '@src/module.api/_core/api.query'
+import { ApiPagedResponse } from '@src/module.api/_core/api.paged.response'
+
+@Controller('/anchors')
+export class AnchorsController {
+  constructor (
+    protected readonly rpcClient: JsonRpcClient
+  ) {
+  }
+
+  /**
+   *  List anchors
+   */
+
+  @Get('')
+  async list (
+    @Query() query: PaginationQuery): Promise<ApiPagedResponse<AnchorData>> {
+    const result = await this.rpcClient.spv.listAnchors({
+      limit: Number(query.size),
+      ...(query.next !== undefined) && { startBTCHeight: Number(query.next) }
```

Comment:
the `spv_listachors` order is from **tip to genesis**
so we can't use `startBTCHeight` as next query

```suggestion
      ...(query.next !== undefined) && { startBTCHeight: Number(query.next) }
      ...(query.next !== undefined) && { maxBtcHeight: Number(query.next) - 1 }
```

Reply:
Not sure if this is gonna work for Pagination. 
https://github.com/DeFiCh/whale/pull/389#discussion_r750995359

Reply:
@canonbrother, your suggestion  `from tip to genesis` help out. Thanks!